defold-spine/editor/src/spineguiext.clj

```diff
@@ -316,33 +323,27 @@
-                (g/connect spine-scenes-node :add-handler-info self :handler-infos)
-                (attach-spine-scene self spine-scenes-node no-spine-scene true)
-                (let [spine-scene-node-prop-keys (g/declared-property-labels SpineSceneNode)
-                      resource-node-prop-keys (g/declared-property-labels gui/ResourceNode)
-                      old-spine-scenes (for [spine-scene-desc (:spine-scenes scene)
-                                             :let [spine-scene-desc (select-keys spine-scene-desc spine-scene-node-prop-keys)]]
-                                         (g/make-nodes graph-id [spine-scene [SpineSceneNode
-                                                                              :name (:name spine-scene-desc)
-                                                                              :spine-scene (workspace/resolve-resource resource (:spine-scene spine-scene-desc))]]
-                                                       (attach-spine-scene self spine-scenes-node spine-scene)))
-                  ;; spine-scene-ext
-                      new-spine-scenes (for [resources-desc (:resources scene)
-                                             :let [resource-desc (select-keys resources-desc resource-node-prop-keys)]]
-                                         (g/make-nodes graph-id [spine-scene [SpineSceneNode
-                                                                              :name (:name resource-desc)
-                                                                              :spine-scene (workspace/resolve-resource resource (:path resource-desc))]]
-                                                       (when (str/ends-with? (:path resource-desc) spineext/spine-scene-ext)
-                                                         (attach-spine-scene self spine-scenes-node spine-scene))))]
-                  (concat old-spine-scenes new-spine-scenes))))
-
+    (g/connect spine-scenes-node :_node-id self :nodes)
+    (g/connect spine-scenes-node :build-errors self :build-errors)
+    (g/connect spine-scenes-node :node-outline self :child-outlines)
+    (g/connect spine-scenes-node :add-handler-info self :handler-infos)
+    (attach-spine-scene self spine-scenes-node no-spine-scene true)
+    (let [spine-scene-name+paths
+          (concat
+            (->> (:spine-scenes scene)
+                 (map (juxt :name :spine-scene)))
+            (->> (:resources scene)
+                 (keep (fn [{:keys [name path]}]
+                         (when (str/ends-with? path spineext/spine-scene-ext)
+                           (pair name path))))))]
+      (for [[name path] spine-scene-name+paths]
+        (let [spine-scene-resource (workspace/resolve-resource resource path)]
+          (g/make-nodes graph-id [spine-scene [SpineSceneNode :name name :spine-scene spine-scene-resource]]
+            (attach-spine-scene self spine-scenes-node spine-scene)))))))
```

Comment:
Indentation + unified code paths for `old-spine-scenes` and `new-spine-scenes`. Removed remnant use of `(g/declared-property-labels gui/ResourceNode)`.